### ISSUE TITLE
allow to change page limit

### ISSUE THREAD

**Issue by jaroslaw-weber** [felixmosh/bull-board#702]
currently the page limit is limited to max 50 pages. but i want to see more. it was unlimited previously. could you add option to edit it when creating a bullboard? (parameter)

**Comment by felixmosh** [comment]
Hi, there is no change to the lib regarding to the number of pages.
Pagination is based on the params that bull / bullmq returns.

Can we debug that somehow?

**Comment by felixmosh** [comment]
There is no such limit...
![image](https://github.com/felixmosh/bull-board/assets/9304194/f64c08a0-489f-479e-9dd0-6af72b8c9d36)

**Comment by felixmosh** [comment]
Closing, feel free to re-open when you have more info

**Comment by jaroslaw-weber** [comment]
sorry i meant 50 jobs PER page. not 50 pages
@felixmosh

**Comment by felixmosh** [comment]
You have a setting for this, no?

**Comment by jaroslaw-weber** [comment]
yes, but i cannot change it to value greater than 50. can do 50 or less

**Comment by felixmosh** [comment]
Make a PR for this?
https://github.com/felixmosh/bull-board/blob/master/packages/ui/src/components/SettingsModal/SettingsModal.tsx#L82
